### PR TITLE
Schema fixes

### PR DESCRIPTION
**Pull request by joshnwalters**

### CODE REVIEW

**Review by mattkim009 — APPROVED** [review]

**Review by mattkim009 — CHANGES_REQUESTED** [review]
Add more info to the defaultProfiles

File: config/settings.development.json [hunk 1]
```diff
@@ -40,6 +42,7 @@
       "firstName": "Matt",
       "lastName": "Kim",
       "website": "https://www.doejohn.com",
+      "skille": "Bioinformatics",
```
Comment (j-taki) [inline config/settings.development.json]: @joshnwalters typo

Comment (mattkim009) [reply config/settings.development.json]: I'll take care of that

**Review by j-taki — APPROVED** [review]

File: config/settings.development.json (hunk 1 above)
Comment (joshnwalters) [reply config/settings.development.json]: fixed skille

**Review by mattkim009 — APPROVED** [review]

**Pull request merged**